src/components/MyListItem/MyListItem.css

```
@@ -0,0 +1,15 @@
+.icon {
+  text-align: center;
+}
+
+td.item {
+  border-width: 0px 1px 0px 1px;
+  border-style: solid solid solid solid;
+  border-color: #000 #000 #000 #000;
+}
+
+td.item.last {
+  border-width: 0px 1px 1px 1px;
+  border-style: solid solid solid solid;
+  border-color: #000 #000 #000 #000;
```

You can use a shorthand
```suggestion
  border-style: solid;
  border-color: #000;
```

---

done